coralproject/talk
# [CORL-1717/20] Visual tweaks + Configurable rec buttons

## What does this PR do?

- Make rec buttons configurable like in normal stream
- border/shadow to differentiate the scrollable vs. non-scrollable region 
- Rec, Reply, and Report styling made to be consistent with one another
- “Report” button should has an active state
- Replace shortcut icon SVG with a font based material icon equivalent

## What changes to the GraphQL/Database Schema does this PR introduce?

None

## How do I test this PR?

Go to live chat stream and see that this is all working and visually looks as expected
